python/ray/ml/predictors/tensorflow.py

```diff
@@ -0,0 +1,60 @@
+import pandas as pd
```

Comment:
Could we put integration-specific functionality in `ray.ml.integrations.*` or `ray.train.integrations.*`? I think cleanly separating integrations from core interfaces is important for readability.